pkg/addon-operator/operator_test.go

```diff
@@ -137,7 +137,7 @@ func assembleTestAddonOperator(t *testing.T, configPath string) (*AddonOperator,
 		MetricStorage:        nil,
 		HookMetricStorage:    nil,
 	}
-	op.ModuleManager = module_manager.NewModuleManager(op.ctx, dirs, &deps)
+	op.ModuleManager = module_manager.NewModuleManager(op.ctx, dirs, &deps, "")
```

Comment:
It seems like a structured config is required form ModuleManager initialization already.